test/java_toolchain/src/main/java/net/thoughtmachine/bazel_compat/BUILD

```diff
@@ -0,0 +1,34 @@
+package(bazel_compatibility = True)
+
+if CONFIG.OS == "linux" or CONFIG.OS == "darwin":
+    please_java_toolchain(
```

Comment:
should these just be java_toolchain? that seems like the more idiomatic name...

Reply:
There's already a java_toolchain rule in Bazel so I have to alias it